app/views/software_choices.scala.html

```diff
@@ -0,0 +1,96 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@(softwareChoices: SoftwareChoicesViewModel)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(title = messages("softwareChoices.title"), bodyClasses = None) {
+    <h1>@messages("softwareChoices.title")</h1>
+
+    <div class="form-group">
+        <p>@messages("softwareChoices.search.text") <a href="#">@messages("softwareChoices.search.text.link")</a>.</p>
+        <div class="panel panel-border-wide">
+            <p>@messages("softwareChoices.text.p1")</p>
+            <p>@messages("softwareChoices.text.p2")</p>
+        </div>
+
+        <div class="form-group search-field search-field-darker">
+            <input type="hidden" name="softwareFilter" value="new">
+            <div class="search-input search-input-with-button">
+                <div class="autocomplete__parent_wrapper">
+                    <div class="autocomplete__wrapper">
+                        <div aria-atomic="true" aria-live="polite" role="status" style="border: 0px; clip: rect(0px 0px 0px 0px); height: 1px; margin-bottom: -1px; margin-right: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 1px;">
```

Comment:
I know this is taken directly from the prototype but I don't like having the inline styles; can we move them to the stylesheet and introduce a class for it.

We need to review the styling in general so that we're not using any custom CSS.